pages/tools.jsx

```diff
@@ -79,8 +79,8 @@ var ToolsSection = React.createClass({
       link: config.THIMBLE,
       src1x: "/img/pages/tools/thimble.png",
       src2x: "/img/pages/tools/thimble@2x.png",
-      activityTitle: "Image Seeking for Fantastic Visual Metaphors",
-      activityLink: "https://cogdog.makes.org/thimble/LTEyMjQ4NjUyOA==/imageseeking-for-fantastic-visual-metaphors"
+      activityTitle: "Keep Calm and Carry On",
+      activityLink: "https://thimble.mozilla.org/anonymous/642982a9-9fa7-43d6-a807-ff3456a8770e/72"
```

Comment:
one of them works, because it's the real link that'll pull up project 72 for remixing, the other one doesn't because it's a personalised link post-remix-load